building/devices/zynqmp.rst

```diff
@@ -69,22 +86,54 @@ Normal World service and run xtest:
 You can close QEMU session at any time by typing ``Ctrl-A+C`` and entering the 
 ``quit`` command.
 
-QEMU for other ZynqMP boards
-******************************
-This makefile supports other ZynqMP boards:
-	
-	* zcu104
-	* zcu106
+Configuring and building for others ZynqMP boards
+*************************************************
+To use this makefile with others supported boards boards, you have to download
+the corresponding BSP and add option ``PLATFORM`` to each make command.
 
-To use this makefile with these boards, you have to download corresponding BSP 
-and add option ``PLATFORM=zcu104`` or ``PLATFORM=zcu106`` to each make command.
+.. code-block:: bash
+	 
+	$ make -f zynqmp.mk PLATFORM=zcu106
+	$ make -f zynqmp.mk PLATFORM=zcu106 qemu
+
+Hereafter the list of available ``PLATFORM``:
+
+	* ``zcu102``
+	* ``zcu104``
+	* ``zcu106``
+	* ``ultra96-reva``
+
+.. warning::
+
+	On Ultra96 board, UART is not directly available. You have to connect
+	through WIFI Access Point using the procedure detailed here 
+	`Getting started`_.
+
+SD card creation
+****************
+After completion of building process, you can create a bootable SD card. Here,
+we consider that SD card corresponds to ``/dev/sdb``. We will use ``gparted``
+and ``e2image`` tool.
+
+Using ``gparted`` or any other partition manager tool create two partions on the
+card:
+
+	* 1GB FAT32 bootable partition (/dev/sdb1 hereafter).
```

Comment:
Prefer double back quotes here and on the next line for (and sdb2) as done on line 115 above.
```rst
``/dev/sdb1``
```